lake/commits/snapshot.go

```diff
@@ -3,6 +3,7 @@ package commits
 import (
 	"errors"
 	"fmt"
+	"golang.org/x/exp/maps"
```

Comment:
We're on Go 1.21 now.
```suggestion
	"maps"
```

Reply:
Huh, the maps packages doesn't have the Keys function :S 

I'll just rewrite that section.